opensrp-reporting/src/main/java/org/smartregister/reporting/impl/models/PieChartViewModel.java

```diff
@@ -0,0 +1,31 @@
+package org.smartregister.reporting.impl.models;
+
+public class PieChartViewModel {
```

Comment:
Also another *DisplayModel*?

Reply:
Exactly its another model for the pie chart view. I created this separately since I realized the binary pie charts have similar implementation like the other models and the **indicator model** would not suffice in this case.